README.md

```diff
@@ -404,7 +435,7 @@ When you are done, your code should look like the following:
             const filtersByParty = {};
             filtersByParty[sender] = { inclusive: { templateIds: [PING, PONG] } };
             const request = {
-                begin: { offsetType: 'boundary', boundary: daml.LedgerOffsetBoundaryValue.END },
+                begin: { offsetType: 'boundary', boundary: ledger.LedgerOffsetBoundaryValue.BEGIN },
                 filter: { filtersByParty: filtersByParty }
```

Comment:
Below, you say

```
    Alice starts reading transactions.
    Created Ping contract from Alice to Bob.
    Transaction read: 1
```

but I actually see

```
Alice starts reading transactions.
Created Ping contract from Alice to Bob.
Transaction read: 0
Transaction read: 1
```

Now, this is due to the fact that I had already ran the code with `createFirstPing()` only. Maybe we should add a note about this, or instructions to reset the sandbox?

Note that you do do this later:
> Before running the application, start with a clean ledger to avoid receiving unprocessed contracts from previous examples:

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/4fc8066183130afd874c1ff334d11e02e98346bf